### DISCUSSION TITLE
formily vue 版本schemafield 版本 下拉框值不显示

### DISCUSSION THREAD

**Discussion by Amoslb** [alibaba/formily#2031] [Q&A 2.x] [answered]
![企业微信截图_9030415a-a80f-4a30-9c7d-cb24d5be8dee](https://user-images.githubusercontent.com/22581560/130039742-b67327af-0fc0-43d9-ba0d-1adea5bf13dc.png)

{
    "type": "object",
    "properties": {
      "j1j1b85cpx1": {
        "title": "Select",
        "x-decorator": "FormItem",
        "x-component": "Select",
        "x-validator": [],
        "x-component-props": {
           "defaultOpen": true
        },
        "x-decorator-props": {},
        "enum": [
          {
            "label": "选项 1",
            "value": "ynm7wswazqg"
          }
        ],
        "_designableId": "j1j1b85cpx1",
        "x-index": 0
      }
    },
  }
    "@formily/vue": "^2.0.0-rc.1",
    "ant-design-vue": "^2.2.6",

**Comment by MisicDemone** [comment]
贴个codesandbox或者吧模板代码也贴一下

**Comment by MisicDemone** [comment, marked as answer]
如果是自己适配的Select组件，需要用connect方法把Field模型上的datasource字段链接到Select组件的options属性

**Reply by Amoslb** [reply]
![image](https://user-images.githubusercontent.com/22581560/130045377-8d2ded04-784f-4ab2-9863-9d90928b3139.png)![企业微信截图_9030415a-a80f-4a30-9c7d-cb24d5be8dee](https://user-images.githubusercontent.com/22581560/130039742-b67327af-0fc0-43d9-ba0d-1adea5bf13dc.png)

![企业微信截图_41c69c2a-6d5d-415e-bd92-f47335691cfe](https://user-images.githubusercontent.com/22581560/130046080-6f86f802-ae15-4e60-992d-2407cec67a40.png)



     select是antd-design-vue中的select 也是需要使用dataSource绑定select吗？
    "@formily/vue": "^2.0.0-rc.1",
    "ant-design-vue": "^2.2.6",
     ruleSchema 值为{
    "type": "object",
    "properties": {
      "j1j1b85cpx1": {
        "title": "Select",
        "x-decorator": "FormItem",
        "x-component": "Select",
        "x-validator": [],
        "x-component-props": {
           "defaultOpen": true
        },
        "x-decorator-props": {},
        "enum": [
          {
            "label": "选项 1",
            "value": "ynm7wswazqg"
          }
        ],
        "_designableId": "j1j1b85cpx1",
        "x-index": 0
      }
    },
  }

**Reply by MisicDemone** [reply]
需要绑定datasource，可以参考@formily/element的适配逻辑

**Reply by Amoslb** [reply]
好的我看一下，谢谢
